sentry_sdk/tracing.py

```diff
@@ -449,11 +449,12 @@ def get_trace_context(self):
 
 
 class Transaction(Span):
-    __slots__ = ("name",)
+    __slots__ = ("name", "parent_sampled")
```

Comment:
Why do we need to store this value in every transaction?

Reply:
Because there's no other way (that I can see) to get it to the `traces_sampler`. The only link between `from_traceparent` and `start_transaction` is the `Transaction` object itself.

It's true that once it's been used, I could delete it off of the transaction object, but is that necessary/worth it?